examples/cra-kitchen-sink/image-storyshots/jest.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
Why is a separate jest config needed? Can it be enough just to use a different extension for image test file, and specify this file explicitly in npm script?